src/components/documents-editor-subtop-nav/DocumentsEditorSubtopNav.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { LineTabs } from "..";
+import { IconChevronDown, IconClose, IconCross, IconOpenSidebarMenu } from "../..";
+import { SDocumentsEditorSubtopNav } from "./styles/SDocumentsEditorSubtopNav";
+import { SDocumentsEditorSubtopNavDocStructure } from "./styles/SDocumentsEditorSubtopNavDocStructure";
+import { SDocumentsEditorSubtopNavIconWrapper } from "./styles/SDocumentsEditorSubtopNavIconWrapper";
+import { SDocumentsEditorSubtopNavTab } from "./styles/SDocumentsEditorSubtopNavTab";
+import { SDocumentsEditorSubtopNavTabsContainer } from "./styles/SDocumentsEditorSubtopNavTabsContainer";
+import { SDocumentsEditorSubtopNavView } from "./styles/SDocumentsEditorSubtopNavView";
+
+interface IDocumentsEditorSubtopNav {
+  documentStructureOpened?: boolean;
+  onOpenDocumentStructure?: (newValue: boolean) => void;
+  blocks: { name: string; id?: string | number }[];
+  selectedBlock: string | number;
+  onSelectBlock: (block: string | number) => void;
+  onViewClick?: () => void;
+}
+
+export const DocumentsEditorSubtopNav: React.FC<IDocumentsEditorSubtopNav> = ({
+  documentStructureOpened,
+  onOpenDocumentStructure,
+  blocks,
+  selectedBlock,
+  onSelectBlock,
+  onViewClick,
+}) => {
+  const handleSelectBlock = (block) => {
```

Comment:
@nicolasshiken why is needed anyways?

Reply:
I've approved the PR, but we may need to remove this since is not doing anything.